### ISSUE TITLE
[Bug]: Prefix & Name of Currency not updating in game chat

### ISSUE THREAD

**Issue by jilienecircelli** [EncryptSL/LiteEco#73]
### What happened?

Hello, 

I have my config file set with a custom prefix and name for the currency which is saved, but in game chat, the prefix and name are set to the default. 

We recently updated to the latest version and it seems it only started after we updated.

![ecolite_ingamechat_issue](https://github.com/EncryptSL/LiteEco/assets/50804429/60b93536-dacf-4c9c-9460-95abd1abc5df)
![ecolite_config_issue](https://github.com/EncryptSL/LiteEco/assets/50804429/0c0229a2-cfbe-4fa4-acf8-36a0b3e9afa7)

### Plugin Version

1.4.1-SNAPSHOT

### You detected problem on this server platform ?

PaperMC

### Version /version - output

```shell
LiteEco version 1.4.1-SNAPSHOT
LiteEco is an economy plugin using VaultAPI
Website: https://encryptsl.cekuj.net
Author: EncryptSQL
Contributors: LcyDev
```

### Relevant log output

```shell
[15:05:32] [Server thread/INFO]: [LiteEco] Enabling LiteEco v1.4.1-SNAPSHOT
[15:05:32] [Server thread/INFO]: [LiteEco] ###################################
[15:05:32] [Server thread/INFO]: [LiteEco] #       PlaceholderAPI Found      #
[15:05:32] [Server thread/INFO]: [LiteEco] #   You can now use placeholders  #
[15:05:32] [Server thread/INFO]: [LiteEco] ###################################
[15:05:32] [Server thread/INFO]: [PlaceholderAPI] Successfully registered internal expansion: liteeco [2.0.5]
[15:05:32] [Server thread/INFO]: [NexEngine] Successfully hooked with LiteEco economy
[15:05:32] [Server thread/INFO]: [LiteEco] ###################################
[15:05:32] [Server thread/INFO]: [LiteEco] # Vault registered like a service #
[15:05:32] [Server thread/INFO]: [LiteEco] ###################################
[15:05:32] [Server thread/INFO]: [LiteEco] ###################################
[15:05:32] [Server thread/INFO]: [LiteEco] #       Treasury not Found        #
[15:05:32] [Server thread/INFO]: [LiteEco] #   For better experience please  #
[15:05:32] [Server thread/INFO]: [LiteEco] #    download treasury or vault   #
[15:05:32] [Server thread/INFO]: [LiteEco] ###################################
[15:05:32] [Server thread/INFO]: [LiteEco] You are using current version !
[15:05:32] [Server thread/INFO]: [LiteEco] Registering commands with Cloud Command Framework !
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener AccountEconomyManageListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener PlayerEconomyPayListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener AdminEconomyGlobalDepositListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener AdminEconomyGlobalSetListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener AdminEconomyGlobalWithdrawListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener AdminEconomyMoneyDepositListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener AdminEconomyMoneyWithdrawListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener AdminEconomyMoneySetListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener PlayerJoinListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Bukkit Listener PlayerQuitListener registered () -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Listeners registered (10) in time 8 ms -> ok
[15:05:32] [Server thread/INFO]: [LiteEco] Plugin enabled in time 607 ms
```

**Comment by EncryptSL** [comment]
Hello dear, @jilienecircelli 

I can't reproduce your issue. Are you sure, you using command /eco reload ?
Your issue is same after save config and restart server ?

**Comment by alec-jensen** [comment]
I also have this issue

**Comment by EncryptSL** [comment]
I still can't reproduce this issue, probably is your issue fixed in 1.4.3-SNAPSHOT

**Download link**: [Click to Update](https://www.spigotmc.org/resources/liteeco-simple-economy-plugin-1-18-x-1-20.101934/update?update=537159)

**Comment by alec-jensen** [comment]
Ok, I was able to fix this issue myself by copying over the config from https://github.com/EncryptSL/LiteEco/blob/main/src/main/resources/config.yml
The config values in the wiki are wrong.
